app/scripts/controllers/overview.js

```diff
@@ -203,6 +203,7 @@ function OverviewController($scope,
            _.size(overview.deployments) +
            _.size(overview.vanillaReplicaSets) +
            _.size(overview.statefulSets) +
+           _.size(overview.daemonSets) +
```

Comment:
Wondering, now that we are up to 8 of these that get repeated in various functions, if we could reduce the duplication.  Noticing its basically the same 8 resources in this first bunch, but `toCheck` only has 7.  Any worry these will get out of sync & cause a bug?  I imagine we would catch it quick so its prob low risk.

Reply:
Agree. I made the same comment to @jwforres 

We might need to backport this particular change, so I'm hesitant to do it in this PR.

The different number between this and the `toCheck` is expected since the latter list shouldn't include service instances. So unfortunately it's not always the same list of resources, which complicates things.

Reply:
That makes sense to me.  Thinking using named arrays/objects would be helpful to make the difference explicit.  

Cool with leaving that for another day.